Wbooks/MainView.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  MainView.swift
+//  Wbooks
+//
+//  Created by Juan Martín Gordo on 06/04/2021.
+//
+
+import Foundation
+import UIKit
+
+final class MainView: NibView {
+    
+    // MARK: Properties
+    @IBOutlet weak var loginButton: UIButton! {
+        didSet {
+            // Modify button properties here
+            
+        }
+    }
```

Comment:
Normally this should be deleted, but since we are going to use it in the next card we can leave it as is.